### DISCUSSION TITLE
Discussion: Is there a repository of custom nodes others have created?

### DISCUSSION THREAD

**Discussion by aceableAlec** [ueberdosis/tiptap#400] [General Questions]
**Discussion**
It's awesome and really helpful that Tiptap lets you customize everything. I'm wondering if anyone has a repository of custom extensions people have made outside of the official spec? I think it would be cool to see people's creations, and maybe helpful to reduce some load on the creators of the project!

**Solution**
I'm not sure what the standard is for these, but just a list of raw links with descriptions and code would be cool. No live demos needed for an MVP of this to plug and play with editors.

**Potential Issue**
If this was a thing, tiptap creators may want to put a disclaimer that they aren't responsible for these extensions, so don't create an issue around them.

**PS**
This is one of my favorite libraries I've ever used and is really well done. Thanks all

**Comment by bbbford** [comment]
I was wondering the same thing! Has anybody improved upon the table node? Would be nice to have it more like the Dropbox Paper table interface...
